docs/flashing.md

```diff
@@ -5,22 +5,39 @@ as appropriate (see [Adafruit's
 documentation](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython),
 though it doesn't cover all CircuitPython boards - you may need to glance around
 the CircuitPython source or ask on Discord). We primarily target CircuitPython
-4.0-alpha1 and above, though many features should work on 3.x. You'll only need
+4.0-alpha1 to 4.0-alpha3. You'll only need
 to flash CircuitPython once (unless we update our baseline supported version).
 
 After CircuitPython has been flashed, a `CIRCUITPY` drive should show up on your
-computer (some Linux/BSD users without drive automounting will want to poke
-around `dmesg` to find the drive identifier and mount this drive manually
-somewhere - ex. `mkdir -p ~/mnt && sudo mount -o uid=1000,gid=1000 /dev/sdf1
-~/mnt`, where `uid` and `gid` are your user ID and primary group ID, as found in
-`id -u` and `id -g`). Take note of the path that this is mounted to (for MacOS
-users, this will probably look something like `/Volumes/CIRCUITPY`).
-
-To "flash" all of KMK, your keymap, and a basic `main.py` that will start
-everything up, run `make MOUNTPOINT=/path/to/wherever
-USER_KEYMAP=path/to/keymap.py`. For example, if my `CIRCUITPY` volume is mounted
-to `~/mnt`, I might flash my development breadboard with the following:
+computer most likely.  If not, check out the troubleshooting section below.
+
+# Windows
+Currently, we do not have an official "flasher" for windows. You can manually install it fairly easily and we recommend coming to the KMK discord/Matrix server if you have any questions. An actual tool is in development. Alternatively, you can flash from any linux like tool set (Cygwin, WSL, ect) using the Linux guide below.
```

Comment:
the matrix community is the only official source of chat/non-Github support. Discord is mentioned in the top-level README which is plenty - it's a hackish bridge I want to discourage except as a last resort.

Reply:
Considering that there are so few users in Matrix, I doubt that if it's not mentioned several times that we support discord will come off as something akin to "You can come ask me how to this linux thing after you install gentoo". It's not so much that we ask a lot, or that the example is reasonable, but we have the bridge for a reason I would assume, and were planning on using it.

Reply:
Added a support page and listed matrix first. If this is acceptable, please mark this is resolved or we can discuss other options.